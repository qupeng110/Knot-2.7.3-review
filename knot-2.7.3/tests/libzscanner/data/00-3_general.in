$ORIGIN	.
$TTL	1

.	NS	@	; No newline