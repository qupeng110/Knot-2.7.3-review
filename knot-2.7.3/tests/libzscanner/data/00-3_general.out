OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
